Jenkinsfile

```
@@ -8,20 +8,22 @@ pipeline {
         timestamps()
     }
     parameters {
-        booleanParam(name: 'unit_validate', defaultValue: true, description: 'amd64 (x86_64) unit tests and vendor check')
-        booleanParam(name: 'amd64', defaultValue: true, description: 'amd64 (x86_64) Build/Test')
-        booleanParam(name: 's390x', defaultValue: true, description: 'IBM Z (s390x) Build/Test')
-        booleanParam(name: 'ppc64le', defaultValue: true, description: 'PowerPC (ppc64le) Build/Test')
+        booleanParam(name: 'unit_validate', defaultValue: false, description: 'amd64 (x86_64) unit tests and vendor check')
+        booleanParam(name: 'amd64', defaultValue: false, description: 'amd64 (x86_64) Build/Test')
+        booleanParam(name: 's390x', defaultValue: false, description: 'IBM Z (s390x) Build/Test')
+        booleanParam(name: 'ppc64le', defaultValue: false, description: 'PowerPC (ppc64le) Build/Test')
         booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
-        booleanParam(name: 'windowsRS5', defaultValue: true, description: 'Windows 2019 (RS5) Build/Test')
-        booleanParam(name: 'skip_dco', defaultValue: false, description: 'Skip the DCO check')
+        booleanParam(name: 'windowsRS5', defaultValue: false, description: 'Windows 2019 (RS5) Build/Test')
+        booleanParam(name: 'skip_dco', defaultValue: true, description: 'Skip the DCO check')
+        booleanParam(name: 'skip_buildkit', defaultValue: false, description: 'Skip the buildkit integration tests')
```

A bit of a "nit", but in hindsight, perhaps I should've named `skip_dco` `dco` for consistency with other stages. In line with that, name this `buildkit`, and enable it by default?

```suggestion
        booleanParam(name: 'buildkit', defaultValue: true, description: 'buildkit integration tests')
```

I'm also thinking of sorting this list alphabetically, so that they match with the position in which they're running in the Jenkins UI

---

Done